code/game/machinery/fusion_engine.dm

```diff
@@ -356,7 +356,7 @@
 			return
 
 		set_overloading(!overloaded)
-		to_chat(user, SPAN_WARNING("You finish [overloaded ? "overloading" : "restoring"] the safeties on [src]."))
+		to_chat(user, SPAN_WARNING("You finish [overloaded ? "restoring" : "overloading"] the safeties on [src]."))
 		log_game("[key_name(user)] has [overloaded ? "overloaded" : "restored the safeties of"] a generator.")
```

Comment:
same here please

Reply:
what are you asking me to do? @kiVts

Reply:
I switched those statements around because the statement checks for if it’s overloaded. previously if it was overloaded it would produce overloading which is wrong so I switched restoring and overloading around.

Reply:
the log